tutorials/getstarted.jl

```diff
@@ -458,7 +458,7 @@ M₈ = SymmetricPositiveDefinite(3)
 # ╔═╡ 82dca58f-52e6-491e-97ec-904bb5b0b36b
 md"""
 which is the manifold of ``3×3`` matrices that are [symmetric and positive definite](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/symmetricpositivedefinite.html#Manifolds.SymmetricPositiveDefinite).
-which has a default as well, the affine invariant [`LinearAffineMetric`](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/symmetricpositivedefinite.html#Default-metric:-the-linear-affine-metric), but also has several different metrics.
+which has a default as well, the affine invariant [`AffineInvariantMetric`](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/symmetricpositivedefinite.html#Default-metric:-the-linear-affine-metric), but also has several different metrics.
```

Comment:
The link is probably incorrect now?

Reply:
Well spotted. I currently can not render the Pluto notebooks with PlutoStaticHTML it seems, but lets see what the CI does.

Reply:
Zes, I am a bit stuck. PlutoStaticHTML fails because it is either not working with Julia 1.9 completely, or because the Pluto notebooks are 1.8.1 notebooks – I am not sure, but I can also not update the hand gestures notebook, since Pluto says that is not a notebook...

Reply:
CI says it should be fine now.